src/ArmTemplates/Commands/Configurations/ExtractorConsoleAppConfiguration.cs

```diff
@@ -98,5 +98,11 @@ public class ExtractorConsoleAppConfiguration
 
         [Option(longName: "overrideProductGuids", HelpText = "Override product GUID identification to system generated")]
         public string OverrideProductGuids { get; set; }
+
+        [Option(longName: "paramApiOauth2Scope", HelpText = "Parametrize API OAuth2 scope values")]
+        public string ParamApiOauth2Scope { get; set; }
+
+        // this is used only from json-file and --extractorConfig option, so no option possibility here
```

Comment:
Normal comment `// comment` is easily lost in the code IMO.
I prefer recommended XML tags for C# methods\properties\fields and etc - [link](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags).

Also when using intellisense you will get that comment right away from opened small window like this (in green circle)
![image](https://user-images.githubusercontent.com/31598696/169808876-58f87d23-0e5e-4fc2-96bd-728bb223ce35.png)

So I suggest
```suggestion
        /// <summary>
        /// OAuth2 scope to override. Available via extractor-config file only.
        /// </summary>
```